Tests/ArgoTests/Models/Post.swift

```diff
@@ -34,7 +34,7 @@ extension LocationPost: Argo.Decodable {
       <*> json["text"]
       <*> json["author"]
       <*> json["comments"]
-      <*> json[optional: "location"]
+      <*> .optional(json["location"])
```

Comment:
These aren't equivalent, right? The new way would be just `<*> json["location"]`?

If `location` has the wrong type, this would fail before but would pass now?

Reply:
Not 100% equivalent, no.

The old behavior (`json[optional: "location"]` or `json <|? "location`) would have failed the decode process if the object under the `location` key wasn't able to be decoded to the type `Location`. But it would have passed if there wasn't any object at that key.

The new behavior (`.optional(json["location"])`) will _always_ succeed, no matter why the decode operation failed.

This is definitely a change in the behavior, but I think it's a simplification that makes it easier to reason about what might fail and for what reasons.

Reply:
But this could just be `<*> json["location"]` to keep the old behavior, right?

Since Argo still has both behaviors, it seems odd to me that you'd change this test. Presumably there are other tests for the `.optional` behavior.